1.9/usage/pods/examples.md

```diff
@@ -4,8 +4,357 @@ feature_maturity: preview
+        "constraints": [],
+        "acceptedResourceRoles": []
+      },
+      "killSelection": "YOUNGEST_FIRST",
+      "unreachableStrategy": {
+        "inactiveAfterSeconds": 900,
+        "expungeAfterSeconds": 604800
+      }
+    },
+    "executorResources": {
+      "cpus": 0.1,
+      "mem": 32,
+      "disk": 10
+    }
+  }
+```
+
+## Additional pod fields
+
+| Field                       | Type     | Value                                                                                                                          |
+|-----------------------------|----------|--------------------------------------------------------------------------------------------------------------------------------|
+| labels                      | object   | Pod metadata as key/value pairs.                                                                                               |
+| environment                 | object   | Environment variables at the pod level. All pod containers will inherit these environment variables. Must be capitalized.      |
+| secrets                     | object   | The fully qualified path to the secret in the store.                                                                           |
+| scheduling                  | object   | Defines exponential backoff behavior for faulty apps in order to prevent sandboxes from filling up.                            |
+| scheduling.backoff               | number   | Initial backoff (seconds) applied when a launched instance fails (default: 1).                                                 |
+| scheduling.backoffFactor         | number   | Factor applied to current backoff to determine the new backoff (default: 1.15).                                                |
+| scheduling.maxLaunchDelay        | number   | Maximum backoff (seconds) applied when subsequent failures are detected (default: 3600).                                       |
+| upgrade                     | object   | Upgrade strategy that controls pod updates.                                                                                    |
+| upgrade.minimumHealthCapacity | number   | Number between 0 and 1 representing the minimum number of healthy nodes to maintain during upgrade (default: 1).               |
+| upgrade.maximumOverCapacity   | number   | Number between 0 and 1 representing the maximum number of additional instances to launch during upgrade (default: 1).          |
+| placement                   | object   | Controls placement of pod tasks.                                                                                               |
+| placement.constraints           | string[] | Constraints control the placement policy of pod tasks. Options: `UNIQUE`, `CLUSTER`, `GROUP_BY`, `LIKE`, `UNLIKE`, `MAX_PER`.  |
+| placement.acceptedResourceRoles | string[] | List of resource roles. Marathon will only consider resource offers with roles on this list for this pod's tasks.              |
```

Comment:
Hmm. It's more precise to say Marathon. I'm going to change it to "The Marathon component....", this way we keep the precision.